tslearn/tests/test_svm.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+from tslearn.metrics import cdist_gak
+from tslearn.svm import TimeSeriesSVC
+
+__author__ = 'Romain Tavenard romain.tavenard[at]univ-rennes2.fr'
+
+
+def test_svm_gak():
```

Comment:
Maybe make the name of the test more explicit (`test_gamma_value_svm`). Would it be worth to also test for other kernels and for SVR (it uses the same base class so maybe not)?

Reply:
OK, Will do